app/web/package.json

```diff
@@ -34,6 +34,7 @@
     "@stripe/stripe-js": "^4.5.0",
     "@toast-ui/editor": "^3.2.2",
     "@types/react-simple-maps": "^3.0.6",
+    "@typescript-eslint/parser": "^8.13.0",
```

Comment:
We were referencing this as the parser in app/web/.eslintrc.json, but didn't have it installed outselves, so it was finding it in the `node_modules` and it was an old version there. Added it so we have control over the version ourselves.